src/extension/app/aem-sidekick.js

```diff
@@ -31,11 +31,16 @@ export class AEMSidekick extends MobxLitElement {
 
     // eslint-disable-next-line no-unused-vars
     this.addEventListener('contextloaded', (data) => {
+      this.requestUpdate();
```

Comment:
If you make `ready` in the siteStore Observable then you won't have to call `requestUpdate`.